site/en/blog/progress-in-the-privacy-sandbox-2021-10/index.md

```diff
@@ -170,23 +171,22 @@ You can track the full [timeline for reducing Chrome's
 user-agent](https://blog.chromium.org/2021/09/user-agent-reduction-origin-trial-and-dates.html),
 with [further examples and details of rollout
 phases](https://www.chromium.org/updates/ua-reduction). You will also need to
-[migrate to User-Agent Client Hints](https://web.dev/migrate-to-ua-ch/) if you
-depend on the platform version, device, or full build version information in the
-current user-agent format.
-
-We're also continuing to make Client Hints overall more consistent with the
-**[Intent to Ship on standardizing existing
-naming](https://groups.google.com/a/chromium.org/g/blink-dev/c/Y42bZ66L6Zo)** by
-adding the `Sec-CH-` header prefix where missing and an **[Intent to Prototype
-on expanding the range of character
-types](https://groups.google.com/a/chromium.org/g/blink-dev/c/ueudFsZzT1M)**
-sent as part of the GREASE mechanism.
+[migrate to User-Agent Client Hints](https://web.dev/migrate-to-ua-ch/) (UA-CH)
+if you rely on the platform version, device, or full build version information
+in the current `User-Agent` format.
+
+We're continuing to [standardize existing names for Client
+Hints](https://groups.google.com/a/chromium.org/g/blink-dev/c/Y42bZ66L6Zo) by
+adding the `Sec-CH-` header prefix where missing. Pending approval, we hope to
```

Comment:
These will be in Chrome 97 beta. It might also be worth mentioning that for the time being, existing headers will be supported for the sake of backward compatibility.

Reply:
That will be addressed in the next progress blog post.